test/src/main/java/org/apache/accumulo/test/mapreduce/TokenFileIT.java

```diff
@@ -84,6 +89,40 @@ protected void cleanup(Context context) throws IOException, InterruptedException
         m.put("", "", Integer.toString(count));
         context.write(new Text(), m);
       }
+
+      @Override
+      protected void setup(Context context) throws IOException, InterruptedException {
+        if (context.getCacheFiles() != null && context.getCacheFiles().length > 0) {
+          // At this point in the MapReduce Job you can get the cached files in HDFS if you want
+          URI[] cachedFiles = context.getCacheFiles();
+          // On the line below we access the file by the pseudonym created during caching in
+          // ConfiguratorBase
+          String fileByPsuedonym = getFileContents(ConfiguratorBase.tokenFileName);
+          assertTrue(!fileByPsuedonym.isEmpty());
+          assertTrue(cachedFiles.length > 0);
+        }
+        super.setup(context);
+      }
+
+      private String getFileContents(String filename) throws FileNotFoundException, IOException {
+
+        StringBuilder sb = new StringBuilder();
+        BufferedReader br = new BufferedReader(new FileReader(filename));
```

Comment:
I think you could read the file contents with many fewer lines of code, especially in Java 8.

Something like:
```java
    return Files.lines(filename).collect(Collectors.joining(System.lineSeparator()));
```